### DISCUSSION TITLE
Improve ChoiceField.choices documentation

### DISCUSSION THREAD

**Discussion by chgad** [encode/django-rest-framework#7835] [Ideas & Suggestions]
Hi, 
i just now stumbled upon that the `ChoiceField` accepts the choices as tuples `(value, display_name)` like documented [here](https://www.django-rest-framework.org/api-guide/fields/#choicefield). As in this issue #4082, i was very surprised that the representation is the `value` not the `display_name`provided in the choices.
So:
1. It would be nice to document this explicitly.
2. Why is the behavior defined this way?

I'd expect the `display_name` as the representation for a given `value` why else should i define the  `display_name` then? The issue mentioned above was briefly answered with a quote form the [django docs](https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.Field.choices)  which, in my eyes, would point towards my expected behavior.
Especially if your API serves a detached frontend app. This app would need the choices-dict too for the correct display of the possible abstract values. Thus leading to code duplication.

**Comment by denolehov** [comment]
You can find choice values and display values in the response from `OPTIONS` (metadata) request.

I would say that serializing choice field values to `value`s instead of `display_name`s makes more sense in the context of APIs, since they're intended to be used programmatically.

If your particular use-case requires `display_name`s, you can inherit choice field and override its `.to_representation()` method.

**Comment by rpkilby** [comment]
@denolehov is correct. The `value` is intended to be used programmatically, while `display_name` is intended for display purposes. Display values aren't suitable for API use, as they may be translated based on the user's locale. 

I've gone into more detail in https://github.com/encode/django-rest-framework/pull/6854#issuecomment-519247839, but in short, it would be nice if there was some construct that differentiated DB values, API representation, and display values.
